apps/emqx_durable_storage/test/emqx_ds_storage_bitfield_lts_SUITE.erl

```diff
@@ -418,6 +481,7 @@ all() -> emqx_common_test_helpers:all(?MODULE).
 suite() -> [{timetrap, {seconds, 20}}].
 
 init_per_suite(Config) ->
+    emqx_common_test_helpers:clear_screen(),
```

Comment:
Forgotten debug?

Reply:
Not anymore: it's a no-op if not explicitly enabled.

https://github.com/emqx/emqx/blob/78d5f76f70a4b5d5d7f83d63ee2bbb45f8c2eb65/apps/emqx/test/emqx_common_test_helpers.erl#L1065-L1080